### PR TITLE
Use `matrix(::MatrixGroupElem)` in more places ; remove `getproperty(::MatrixGroupElem, sym)` code for `sym == :elm`

### PR DESCRIPTION
**Pull request by fingolfin**

### CODE REVIEW

**Review by lgoettgens — APPROVED** [review]
Looks good

**Comment by lgoettgens** [comment]
I think you need to run the auto-formatter on some of the files where the Formatter CI job is enabled on.

File: experimental/LinearQuotients/src/cox_rings.jl [hunk 1]
```diff
@@ -285,7 +285,7 @@ function cox_ring(
   # to the grading induced by the action of A.
 
   # The action of A on RH
-  A_action = (f, g) -> right_action(f, preimage(GtoA, g).elm)
+  A_action = (f, g) -> matrix(right_action(f, preimage(GtoA, g)))
```
Comment (joschmitt) [inline experimental/LinearQuotients/src/cox_rings.jl]:
```suggestion
  A_action = (f, g) -> right_action(f, matrix(preimage(GtoA, g)))
```

**Review by joschmitt — APPROVED** [review]
Thank you!

**Pull request merged**